jvb/src/main/kotlin/org/jitsi/videobridge/octo/config/OctoConfig.kt

```diff
@@ -17,40 +17,12 @@
 package org.jitsi.videobridge.octo.config
 
 import org.jitsi.config.JitsiConfig
-import org.jitsi.metaconfig.ConfigException
 import org.jitsi.metaconfig.config
 import org.jitsi.metaconfig.from
 import org.jitsi.metaconfig.optionalconfig
 
 class OctoConfig private constructor() {
```

Comment:
Seems like it could be good to rename this, and maybe even rename the config paths to be consistent? So the old-old config paths are removed completely, the current config paths which use the `octo` scope are still supported, but new paths (scoped around `relay` or whatever) are now used as defaults?